README.md

```diff
@@ -48,6 +48,59 @@ made available on the GitHub releases page: https://github.com/MilesCranmer/rip2
 
 To install, simply open the archive and move the binary somewhere you can run it.
 
+<details>
+<summary>Nix</summary>
```

Comment:
```suggestion
<summary>Additional nix options</summary>
```

And then add a normal header/description before the `<details>` block, similar to what is done for Cargo and the binaries. Like

```
### Nix flake

(Short description, maybe show the shell command example for quickly running it)

(then, the further details wrapped in a toggle)
```